### ISSUE TITLE
Bug: failing ggml.c:12621: ne2 == ne02 during finetuning 

### ISSUE THREAD

**Issue by hgftrdw45ud67is8o89** [ggerganov/llama.cpp#8564]
### What happened?

win11,nvidia gpu
uses llama.cpp with cuda 12

ran finetuning with
`llama-finetune --model-base ...\Aura-Uncensored-OAS-8B-L3.Q5_K_M.gguf --train-data ".......\phpgood practices_cleaned.txt" --threads 30 --save-every 5 --epochs 5 --adam-iter 128 --warmup 10 -c 128 --checkpoint-out test_lora_k_m-ITERATION.gguf --grad-acc 2`

tried with different model but has same result.

### Name and Version

version: 3400 (97bdd26e)
built with MSVC 19.29.30154.0 for x64

### Relevant log output

```shell
tokenize_file: total number of samples: 676
main: number of training tokens: 804
main: number of unique tokens: 377
main: train data seems to have changed. restarting shuffled epoch.
main: begin training
main: work_size = 15392728 bytes (14.7 MB)
train_opt_callback: iter=     0 sample=1/676 sched=0.000000 loss=0.000000 |->
GGML_ASSERT: D:\a\llama.cpp\llama.cpp\ggml\src\ggml.c:12621: ne2 == ne02
```

**Comment by hgftrdw45ud67is8o89** [comment]
RE-issue not fixed.
